notes/stv/20201007-AXS.txt

```diff
@@ -0,0 +1,75 @@
+sudo yum install wget
```

Comment:
Am I right in thinking these notes are testing the AXS deploy by manually installing a modified version of Hadoop from the AXS github repository - not part of the ansible deployment.
Do we have a test that demonstrates what features the modified version of Hadoop provides that is not available in the standard Hadoop?

Reply:
It would be useful to reference how the cluster is configured.
If this is following on from previous notes, then just a one line comment referencing them would help to understand the context.
Are these steps being run on a master or worker node, a single node or all of them ?